lib/layout.js

```diff
@@ -113,6 +114,27 @@ const PodiumLayout = class PodiumLayout {
             value: new Metrics(),
         });
 
+        // Skip a tick to ensure the metric stream has been consumed
+        setImmediate(() => {
+            const moduleVersion = pkg.version;
+            const segments = moduleVersion.split('.').map(value => {
+                return parseInt(value, 10);
+            });
+
+            this.metrics.metric({
```

Comment:
in keeping with new metrics syntax, this should really be:

```js
// defined somewhere
const versionGauge = this.metrics.gauge({
    name: 'podium_layout_version_info',
    description: '@podium/layout version info',
);

// used when collecting the metric
versionGauge.set(1, {
    labels: { 
        version: moduleVersion,
        major: segments[0],
        minor: segments[1],
        patch: segments[2],
    }
});
```

Reply:
+1 on this.